packages/core/admin/admin/src/content-manager/pages/EditSettingsView/components/LayoutDndProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import type { EditSettingsViewState } from './SettingsForm/reducer';
+import type { Schema } from '@strapi/types';
+
+export interface LayoutDndProviderProps {
+  isContentTypeView: boolean;
+  attributes: Schema.Attributes;
+  modifiedData: EditSettingsViewState['modifiedData'];
+  slug: string;
+  componentLayouts: EditSettingsViewState['componentLayouts'];
+  selectedField: string;
+  fieldForm: any;
+  moveRow: (fromIndex: number, toIndex: number) => void;
+  moveItem: (
+    dragIndex: number,
+    hoverIndex: number,
+    dragRowIndex: number,
+    hoverRowIndex: number
+  ) => void;
+  setEditFieldToSelect: (name: string) => void;
+  isDraggingSibling: boolean;
+  setIsDraggingSibling: (isDragging: boolean) => void;
+  children?: React.ReactNode;
+}
+
+const LayoutDndContext = React.createContext<LayoutDndProviderProps>(
```

Comment:
```suggestion
const LayoutDndContext = React.createContext<Omit<LayoutDndProviderProps, 'children'>>(
```

Children isn't available on the context